golang/cosmos/app/upgrade.go

```diff
@@ -174,55 +182,47 @@ func unreleasedUpgradeHandler(app *GaiaApp, targetUpgrade string) func(sdk.Conte
 
 			// Each CoreProposalStep runs sequentially, and can be constructed from
 			// one or more modules executing in parallel within the step.
+			CoreProposalSteps = append(CoreProposalSteps,
+            	// because of #10794, we need to do at least a null upgrade of
+            	// the walletFactory on every software upgrade
```

Comment:
Looks like the comments needs formatting (weird indentation)

Reply:
done